.github/workflows/tlaplus.yml

```diff
@@ -0,0 +1,15 @@
+name: "TLA+ Spec Testing"
+
+on: [push, pull_request]
+
+jobs:
+
+  model-checker:
+    name: Run TLC Model Checker Against CCF Raft Spec
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - run: python ./tla/install_deps.py
+      - run: (cd ./tla && tlc.sh MCccfraft.tla)
```

Comment:
The run section should also include the second model, which takes about 35s on my local machine.  Assuming a sensible `tlc` alias: 

```
tlc -config MCccfraft.cfg MCccfraftWithReconfig.tla
```

Reply:
It might be better to use the `tlc` alias from `bashrc` instead of the `tlc.sh` wrapper. They should both work but they pass slightly different configuration parameters to java and I assume the `tlc` alias ones are better (because they were written by @lemmy instead of by me :-))